BingMaps/rest-services/examples/driving-route-with-route-path-example.md

```diff
@@ -16,7 +17,7 @@ ms.service: "bing-maps"
 
 # Driving Route with Route Path Example
 
-This example returns a driving route from the Minneapolis, Minnesota to St. Paul, Minnesota. The route is optimized for distance and requests point values for the route’s path. Results are shown for both JSON and XML response formats..  
+This example returns a driving route from the Minneapolis, Minnesota to St. Paul, Minnesota. The route is optimized for distance and requests point values for the route’s path. Results are shown for both JSON and XML response formats.  
```

Comment:
his example returns a driving route from Minneapolis, Minnesota to St. Paul, Minnesota.